master/getting-started/bare-metal/bare-metal.md

```diff
@@ -427,3 +430,280 @@ an ingress rule allowing access *to* port 999 and an egress rule allowing
+in terms of a packet's original destination IP address and port, than in terms
+of that packet's destination IP address and port after it has been DNAT'd.
+
+An example is securing access to Kubernetes NodePorts from outside the cluster.
+Traffic from outside is addressed to any node's IP address, on a known
+NodePort, and Kubernetes (kube-proxy) then DNATs that to the IP address of one
+of the pods that provides the corresponding Service, and the relevant port
+number on that port (which is usually different from the NodePort).
+
+As NodePorts are the externally advertised way of connecting to Services (and a
+NodePort uniquely identifies a Service, whereas an internal port number may
+not), it makes sense to express Calico policy to expose or secure particular
+Services in terms of the corresponding NodePorts.  But that is only possible if
+the Calico policy is applied before DNAT changes the NodePort to something
+else - and hence this kind of policy needs 'preDNAT' set to true.
+
+In addition to being applied before any DNAT, the enforcement of pre-DNAT
+policy differs from that of normal host endpoint policy in three key details,
+reflecting that it is designed for the policing of incoming traffic from
+outside the cluster:
+
+1. Only the ingress rules of a pre-DNAT policy are enforced.  If the policy has
+   any egress rules defined, they will have no effect.  (Standard connection
+   tracking is sufficient to allow return path traffic.)
+
+2. Pre-DNAT policy is enforced for all traffic arriving through a host
+   endpoint, regardless of where that traffic is going, and - in particular -
+   even if that traffic is routed to a local workload on the same host.
+   (Whereas normal host endpoint policy is skipped, for traffic going to a
+   local workload.)
+
+3. There is no 'default drop' semantic for pre-DNAT policy (as there is for
+   normal host endpoint policy).  In other words, if a host endpoint is defined
+   but has no pre-DNAT policies that explicitly allow or deny a particular
+   incoming packet, that packet is allowed to continue on its way, and will
+   then be accepted or dropped according to workload policy - if it is going to
+   a local workload - or to normal host endpoint policy - if not.
+
+## When do host endpoint policies apply?
+
+As stated above, normal host endpoint policies apply to traffic that arrives on
+and/or is sent to a host interface, except if that traffic comes from or is
+destined for a workload on the same host; but the rules for applying untracked
+and pre-DNAT policies are different in some cases.  Here we present and
+summarize all of those rules together, for all possible flows and all types of
+host endpoints policy.
+
+For packets that arrive on a host interface and are destined for a local
+workload - i.e. a locally-hosted pod, container or VM:
+
+- Untracked policies technically do apply, but never have any net positive
```

Comment:
To stay consistent, I'll move all of the (effectively-)non-applying cases to the end of their sections.  I thought it was nice to have the really-applying ones in the order that they happen, though.